src/main/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurer.java

```diff
@@ -169,17 +176,28 @@ void updateAdminPassword() {
                 System.exit(-1);
             }
 
-            // Print an update to the logs
-            System.out.println("Admin password set successfully.");
-
+            // Update the custom password in internal_users.yml file
             writePasswordToInternalUsersFile(ADMIN_PASSWORD, INTERNAL_USERS_FILE_PATH);
 
+            System.out.println("Admin password set successfully.");
```

Comment:
Do you know why we are using system.print instead of loggers like in the rest of the code?

Reply:
this was done to follow suit with other tools. I have at some point tried logger while rewriting this tool but had run into issues. We can track this in a separate PR.